cashweb-registry/src/store/db.rs

```diff
@@ -15,6 +15,7 @@ pub(crate) const CF_METADATA: &str = "metadata";
 pub(crate) const CF_PKH_BY_TIME: &str = "pkh_by_time";
 pub(crate) const CF_MESSAGES: &str = "topic_messages";
 pub(crate) const CF_PAYLOADS: &str = "message_payloads";
+pub(crate) const CF_BURNS: &str = "topic_burn_txs";
```

Comment:
```suggestion
pub(crate) const CF_TOPIC_BURNS: &str = "topic_burn_txs";
```
there might be other burn txs; this kinda implies this stores all kinds of burn txs